bool set_sleep
---
bool succeed
